test/sim/misc/target-resolution.js

```diff
@@ -323,4 +323,23 @@ describe('Target Resolution', function () {
 		battle.makeChoices('auto', 'move phantomforce 1, move sheercold 1');
 		assert.false.fullHP(battle.p1.active[1], 'Altaria should not be at full HP, because Phantom Force was redirected and targeted it.');
 	});
+
+	it.skip(`should cause Rollout to target the same slot after being called as a submove`, function () {
+		// hardcoded RNG seed to show the erroneous targeting behavior
+		battle = common.createBattle({gameType: 'doubles', seed: [1, 2, 3, 4]}, [[
+			{species: 'purrloin', ability: 'compoundeyes', moves: ['rollout', 'sleeptalk']},
+			{species: 'regieleki', moves: ['healbell', 'spore']},
+		], [
+			{species: 'aggron', moves: ['sleeptalk']},
+			{species: 'slowbro', moves: ['sleeptalk']},
+		]]);
+
+		battle.makeChoices('move sleeptalk, move spore -1', 'auto');
```

Comment:
This test is wrong, as sleep prevents Rollout from locking. I am working on this bug so I will write a new test using Copycat instead.